src/server/services/wms/qgswmsparameters.cpp

```diff
@@ -688,7 +688,7 @@ namespace QgsWms
     return version;
   }
 
-  double QgsWmsParameters::toDouble( QVariant value, QVariant defaultValue, bool *error ) const
+  double QgsWmsParameters::toDouble( QVariant value, const QVariant &defaultValue, bool *error ) const
```

Comment:
The `QVariant value` parameter should actually be `const QVariant &value` throughout this file